components/DataTable.js

```diff
@@ -1,36 +1,127 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import Link from 'next/link';
 
 const DataTable = ({ datasets }) => (
-  <Table>
-    <thead>
-      <tr>
-        <th>Data Set</th>
-        <th>Last Updated</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody>
+  <Container>
+    <div className="headers">
+      <div className="first-col">Data set</div>
+      <div className="second-col">Last updated</div>
+    </div>
+    <div className="datasets">
       {datasets.map((dataset, index) => (
-        <tr key={index}>
-          <td>
-            <h4>{dataset.title}</h4>
-            <p>{dataset.description}</p>
-          </td>
-          <td>{dataset.date}</td>
-          <td>
-            <a href={dataset.link}>View</a>
-          </td>
-        </tr>
+        <div className="dataset" key={index}>
+          <div className="first-col">
+            <h4 className="dataset-title">{dataset.title}</h4>
+            <div className="dataset-date">{dataset.date}</div>
+            <p className="dataset-description">{dataset.description}</p>
+          </div>
+          <div className="second-col dataset-date-md">{dataset.date}</div>
+          <Link href={dataset.link}>
+            <a className="dataset-link">View</a>
+          </Link>
+        </div>
       ))}
-    </tbody>
-  </Table>
+    </div>
+  </Container>
 );
 export default DataTable;
 
 DataTable.propTypes = {
-  datasets: PropTypes.object.isRequired,
+  datasets: PropTypes.array.isRequired,
 };
 
-const Table = styled.table``;
+const Container = styled.div`
+  padding: 2rem 0;
+
+  @media (min-width: ${props => props.theme.medium}) {
+    .first-col {
+      width: 65%;
+      margin-right: 3rem;
+    }
+
+    .second-col {
+      width: 20%;
+      margin-right: 1.5rem;
+    }
+  }
+
+  .headers {
+    display: none;
+    font-weight: ${props => props.theme.fontWeights.bold};
+    text-transform: uppercase;
+
+    @media (min-width: ${props => props.theme.medium}) {
+      display: flex;
+      margin: 1.25rem 0;
+    }
+  }
+
+  .datasets {
+    .dataset:first-child {
+      border-top: 6px solid ${props => props.theme.colors.primaryBlue};
+    }
+
+    .dataset {
+      border-top: 1px solid ${props => props.theme.colors.gray};
+      padding: 3rem 0;
+
+      @media (min-width: ${props => props.theme.medium}) {
+        display: flex;
+      }
+
+      .dataset-title {
+        color: ${props => props.theme.colors.black};
+        font-weight: ${props => props.theme.fontWeights.bold};
+        letter-spacing: ${props => props.theme.letterSpacings.medium};
+
+        @media (min-width: ${props => props.theme.medium}) {
+          font-size: 22px;
```

Comment:
I haven't looked into the Figma files much, so I just want to make sure these font sizes align with our existing sizes well. Should we go set these up as variables?

I know we have talked about Julia doing a full audit of the size to help establish a more uniform style guide, so maybe font sizes can be included in that if we are starting to build up a lot of variation.

Reply:
I like that idea! Not sure if these are the sizes we'll want to stick with, but in https://github.com/texas-justice-initiative/website-nextjs/pull/471/commits/f6544c4abc692e38a543ec5d51fe4e197905864a, I added variables for the different font sizes and line heights I saw in Figma, using the values we use for our normal body text as the `md` size for both. What do you think?

Reply:
Yeah I think this is great @nholden! We have them available and can address the sizes more easily this way. I like the refactor on the variables!